src/Services/PaymentService.php

```
@@ -166,6 +166,28 @@ public function restore(string $id): array
+     * 
+     * @param  string  $id  The ID of the payment whose status is to be retrieved.
+     * @return array An array containing the status of the specified payment.
+     * 
+     * @throws AuthenticationException
+     * @throws NotFoundException
+     * @throws RateLimitException
+     * @throws ApiException
+     * @throws ValidationException
+     * @throws \InvalidArgumentException if the provided ID is empty.
+     */
+    public function getStatus(string $id): array
```

PHPStan (2.1.31)

Method AsaasPhpSdk\Services\PaymentService::getStatus() return type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)